march_hardware/include/march_hardware/EtherCAT/EthercatIO.h

```diff
@@ -5,7 +5,8 @@
 #include <stdint.h>
 #include <stdio.h>
 
-extern "C" {
+extern "C"
+{
 #include "../../../soem/ethercat.h"
```

Comment:
Can this be done differently?

Reply:
Not very cleanly unless we move the `ethercat.h` to the include folder maybe?

Reply:
Haven't been able to find a solution for this. It's only needed when the hardware_interface includes the library, not when running standalone, not sure why.

Reply:
OK for this PR, maybe something to look into in the future.